apis/events/protobuf/v1/event.proto

```diff
@@ -186,3 +193,11 @@ message HostInitiatorEventDetails {
   // Target uuid
   string uuid = 4;
 }
+
+// Reactor event details
+message ReactorEventDetails {
```

Comment:
Frozen reactors can also unfreeze, probably need to take into account here so we can get events for both freeze and unfreeze?

Reply:
Or maybe just have a ReactorUnfreeze above